unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -2900,64 +2936,69 @@ public struct WhatDidITouch {
             return result;
         }
 
-        // rotat ethe hand if there is an object in it
-        public void RotateHand(ServerAction action) {
+        [ObsoleteAttribute(message: "This action is deprecated. Call RotateHeldObject() instead.", error: false)]
+        public void RotateHand(float x, float y, float z) {
+            RotateHeldObject(rotation: new Vector3(x, y, z));
+        }
 
+        // rotate the held object if there is an object in it
+        public void RotateHeldObject(Vector3 rotation) {
             if (ItemInHand == null) {
-                errorMessage = "Can't rotate hand unless holding object";
-                actionFinished(false);
-                return;
+                throw new InvalidOperationException("Can't rotate hand unless holding object.");
```

Comment:
with the semantic change from "hand" to "held object" might as well change this exception message as well. Something like "Can't rotate object. No object held."

Reply:
Updated in https://github.com/allenai/ai2thor/pull/770/commits/dcc9ef823c6793f49dba3c0b7a2f1027f5096a1b.